everware/spawner.py

```diff
@@ -412,6 +412,7 @@ def get_env(self):
             env.update({
                 'JPY_GITHUBURL': self.repo_url_with_token,
                 'JPY_REPOPOINTER': self.commit_sha,
+                'API': 'api_from_openml'
```

Comment:
it can't help , yet. But api-key is send from the Openml site and have to be set in the container for the connecting with OpenMl (http://openml.github.io/openml-python/stable/usage.html). So with handling of get-request , we will set API variable and push it in the container like this.